### PR TITLE
Meag rewordings

### PR DESCRIPTION
**Pull request by meaghanfitzgerald**
Added explanations to the options for creating a subnet with the cli

### CODE REVIEW

File: cmd/subnetcmd/create.go [hunk 1]
```diff
@@ -482,7 +478,7 @@ func createSubnetConfig(cmd *cobra.Command, args []string) error {
 				options := []string{anyoneCanSubmitTransactionsOption, approvedCanSubmitTransactionsOption, explainOption}
 				for {
 					option, err := app.Prompt.CaptureList(
-						"Do you want to allow only certain addresses to interact with your blockchain? (Transaction Allowlist Precompile)",
+						"Do you want to allow only certain user addresses to interact with your blockchain? (Transaction Allowlist Precompile)",
```
Comment (meaghanfitzgerald) [inline cmd/subnetcmd/create.go:481]:
```suggestion
						"Do you want to allow only certain user addresses to interact with your blockchain? (Transaction Allow List Precompile)",
```

File: cmd/subnetcmd/create.go [hunk 2]
```diff
@@ -378,10 +379,7 @@ func createSubnetConfig(cmd *cobra.Command, args []string) error {
 				if err != nil {
 					return err
 				}
-			case explainOption:
-				ux.Logger.PrintToUser("The difference is...")
-				ux.Logger.PrintToUser("")
-				continue
+				//missing case for dontChangeFeeSettingsOption
```
Comment (meaghanfitzgerald) [inline cmd/subnetcmd/create.go:382]: there is a missing case for the `dontChangeFeeSettingOption`

Comment (felipemadero) [reply cmd/subnetcmd/create.go:382]: yup. working on the implementation. thanks for this!

**Review by felipemadero — APPROVED** [review]
LGTM!

**Pull request merged**